integration_tests/README.md

```diff
@@ -83,6 +83,30 @@ As an example, here is the `spark-submit` command with the TPCxBB parameters:
 $SPARK_HOME/bin/spark-submit --jars "rapids-4-spark_2.12-0.3.0-SNAPSHOT.jar,cudf-0.16-SNAPSHOT.jar,rapids-4-spark-tests_2.12-0.3.0-SNAPSHOT.jar" ./runtests.py --tpcxbb_format="csv" --tpcxbb_path="/path/to/tpcxbb/csv"
 ```
 
+### Enabling cudf_udf Tests
+
+The cudf_udf tests in this framework are testing Pandas UDF(user-defined function) with cuDF. They are disabled by default because of complicated environment setup and can be enabled by providing option:
+
+   * `cudf_udf` (optional, defaults to "False")
+
+cudf_udf tests needs a couple of different settings, they may need to run separately.
+
+To enable cudf_udf tests, need following pre requirements:
+   * Install cuDF library on all the nodes running executors. The instruction could be found at [here](https://rapids.ai/start.html). Please follow the steps to choose the version based on your environment and install the cuDF library via Conda or use other ways like building from source.
+   * Disable the GPU exclusive mode on all the nodes running executors. The sample command is `sudo nvidia-smi -c DEFAULT`
```

Comment:
why do we need this?  Is it because python uses a different gpu than java? We should state that the node needs at least 2 gpus in that case and state why we need to turn it off.  This also implies it can't be run on yarn in isolated mode  so we should specify the environment it needs to run in. for instance if only local mode supported

Reply:
Right now we share the GPU between the java process and the python process.  That can be changed in the future, but would require changes with how the plugin does initialization, which we didn't want to get into for the first round of the feature.  It may come later.  So yes it does mean that we cannot run with this feature on a yarn cluster in isolated mode.

Reply:
Thanks Bobby for explaining.